adapter-http/src/main/resources/activities/baselines/http-rest-keyvalue.yaml

```diff
@@ -15,9 +15,9 @@ scenarios:
 
 bindings:
 
-  weighted_hosts: ToString(); WeightedStrings('<<restapi_host:stargate>>'); ToString();
+  weighted_hosts: ToString(); WeightedStrings('<<stargate_host:stargate>>'); ToString();
   request_id: ToHashedUUID(); ToString();
-  auto_gen_token: Discard(); StargateToken('http://localhost:8081/v1/auth'); ToString();
+  token: Discard(); Token('<<auth_token:>>','<<uri:http://localhost:8081/v1/auth>>', '<<uid:cassandra>>', '<<pswd:cassandra>>');
```

Comment:
Nit: perhaps `auth_uri`, `auth_uid`, `auth_pswd` would be more descriptive variable names?